xbzu1/1.0/board.xml

```diff
@@ -60,37 +61,247 @@
+                <port_map logical_port="SDA_O" physical_port="TempSensor_i2c_main_sda_o" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SENSOR_I2C_SDA"/>
+                  </pin_maps>
+                </port_map>
+                <port_map logical_port="SDA_T" physical_port="TempSensor_i2c_main_sda_t" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SENSOR_I2C_SDA"/>
+                  </pin_maps>
+                </port_map>
+              </port_maps>
+            </interface>
+
+            <!-- SYZYGY DNA I2C -->
+    	    <interface mode="master" name="SyzygyDNA_i2c_pl" type="xilinx.com:interface:iic_rtl:1.0" of_component="SyzygyDNA_i2c_pl">
+	          <preferred_ips>
+                <preferred_ip vendor="xilinx.com" library="ip" name="axi_iic" order="0"/>
+              </preferred_ips>
+              <port_maps>
+                <port_map logical_port="SCL_I" physical_port="SyzygyDNA_i2c_main_scl_i" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SYZYGY_SCL_1V8"/>
+                  </pin_maps>
+                </port_map>
+                <port_map logical_port="SCL_O" physical_port="SyzygyDNA_i2c_main_scl_o" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SYZYGY_SCL_1V8"/>
+                  </pin_maps>
+                </port_map>
+                <port_map logical_port="SCL_T" physical_port="SyzygyDNA_i2c_main_scl_t" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SYZYGY_SCL_1V8"/>
+                  </pin_maps>
+                </port_map>
+                <port_map logical_port="SDA_I" physical_port="SyzygyDNA_i2c_main_sda_i" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SYZYGY_SDA_1V8"/>
+                  </pin_maps>
+                </port_map>
+                <port_map logical_port="SDA_O" physical_port="SyzygyDNA_i2c_main_sda_o" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SYZYGY_SDA_1V8"/>
+                  </pin_maps>
+                </port_map>
+                <port_map logical_port="SDA_T" physical_port="SyzygyDNA_i2c_main_sda_t" dir="inout">
+                  <pin_maps>
+                    <pin_map port_index="0" component_pin="HD_SYZYGY_SDA_1V8"/>
+                  </pin_maps>
+                </port_map>
+              </port_maps>
+            </interface>
+
+            <!-- SYZYGY STD PL (J6) -->
+
+            <!-- SYZYGY TRX2 PL (J1) -->
+
+	  	    <!-- SYZYGY TRX2 MIO (J2) -->
+
+			</interfaces> 
       </component>
 
+      <!-- ZU+ PS Fixed I/O -->
       <component name="ps8_fixedio" display_name="PS8 fixed IO" type="chip" sub_type="fixed_io" major_group=""/>
-   </components>
```

Comment:
Removing this line looks like an error. It will mean, in xml speak, the tag was "opened" (<>)but never "closed"(</>)

Reply:
It's on line 255